swagger.yaml

```diff
@@ -183,8 +185,12 @@ paths:
           type: string
         - name: source_metadata
           in: query
-          description: '???'
+          description: >-
+            A JSON blob containing extra source specific information.  This
+            field is free-form, but we generally expect the data will follow
+            one of the formats outlined in the `/schema` folder.
```

Comment:
Oh! Actually, you can rename `swagger.yml` to `swagger.yml.erb` and then have a description field like:

```yml
- description: >-
  Here is a link to
  <a href="<%= asset_path('versionista.json') %>">the Versionista schema</a>.
```